### ISSUE TITLE
Cannot run fl training on cora in distributed mode

### ISSUE THREAD

**Issue by SXxtyz** [alibaba/FederatedScope#489]
FedSagePlusServer process has no response when clients send gen_para

**Comment by xieyxclack** [comment]
Could you provide more details to reproduce this issue, thx!

**Comment by SXxtyz** [comment]
Here is the cfg file of client_1, it seems that when the client completes the pretrain and send msg_type='gen_para',  comm_manager.receive() doesn't listen msg_type='gen_para' in Server.run()
```yaml
use_gpu: True
federate:
  client_num: 3
  mode: 'distributed'
  total_round_num: 100
  make_global_eval: True
  online_aggr: False
  method: fedsageplus
distribute:
  use: True
  server_host: '127.0.0.1'
  server_port: 50051
  client_host: '127.0.0.1'
  client_port: 50052
  role: 'client'
  data_idx: 1
trainer:
  type: nodefullbatch_trainer
eval:
  metrics: [ 'acc', 'correct' ]
data:
  root: /data1/zdj/bs/dataset/
  type: 'cora'
  splitter: 'louvain'
model:
  type: sage
  hidden: 64
  dropout: 0.5
  out_channels: 7
dataloader:
  type: pyg
  batch_size: 1
fedsageplus:
  num_pred: 5
  gen_hidden: 64
  hide_portion: 0.5
  fedgen_epoch: 20
  loc_epoch: 1
  a: 1.0
  b: 1.0
  c: 1.0
train:
  batch_or_epoch: epoch
criterion:
  type: 'CrossEntropyLoss'
